sample-apps/segmentation/main.py

```diff
@@ -80,3 +88,27 @@ def init_strategies(self):
             "random": Random(),
             "first": MyStrategy(),
         }
+
```

Comment:
Lets discuss... probably we want to avoid this for public release.. and keep all generic scribbles for all apps that doesn't have app specific dependency (in this case infer output from other infer tasks)

Reply:
Sure, lets discuss in our meeting. This infer function works just fine for non scribbles tasks. The dependency is broken when we try to run scribbles method that need logits from CNN. Maybe there is a more elegant way of achieving this....

Reply:
Another thought, maybe we can only keep coldstart for this generic app - which wont require logits. Then perhaps we may want to keep atleast one app around that covers/showcases all scribbles functionality (e.g. **segmentation_spleen_scribbles** that is due to be removed by end of this week)